18/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int k, a[100], n, c = 0;
+
+    cin >> n;
+
+    for(int i = 0; i < n; i++){
+        cin >> a[i];
+    }
+
+    cin >> k;
+
+    bool gasit = false;
+
+    for(int i = 0; i < n && gasit == false; i++){
```

Comment:
Ca si info aici, in loc e expresia "gasit == false" poti folosi expresia !gasit. Semnul exclamarii este folosit pentru negatie
if (!gasit) echivalent cu if (gasit == false)
if (gasit) echivalent cu if (gasit == true)